extensions/objects/templates/image-with-regions-editor.component.ts

```diff
@@ -314,6 +314,15 @@ export class ImageWithRegionsEditorComponent implements OnInit {
   }
 
   getPreviewUrl(imageUrl: string): string {
+    const entityType: string = this.contextService.getEntityType() as string;
```

Comment:
This should work the same
```suggestion
    const entityType = this.contextService.getEntityType() as string;
```